app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Contestants Index Page</h1>
+
+<% @contestants.each do |contestant|%>
+
+    <h3><%= "Name: #{contestant.name}" %></h3>
+    <h3><%= "Projects:" %></h3>
+        <% contestant.projects.each do |project| %>
+            <h3><%= "#{project.name}" %></h3>
+        <% end %>
```

Comment:
You don't have to string interpolate in here and you can do a nested iteration like you have but you could have also written a method in your `Contestant` model to get the projects. So something like:
```ruby
def projects_list
 projects.pluck(:name).to_sentence
end
```
Then instead of iterating again on line 7, you could just call
```ruby
contestant.projects_list
```

Reply:
This is a way better way of doing it! Thank you, definitely going to be looking for ways to clean up my views like this.